veniq/dataset_collection/augmentation.py

```diff
@@ -54,12 +58,13 @@ def get_line_with_first_open_bracket(
         file_path: Path,
         method_decl_start_line: int
 ) -> int:
-    f = open(file_path, encoding='utf-8')
-    file_lines = list(f)
-    for i, line in enumerate(file_lines[method_decl_start_line - 2:], method_decl_start_line - 2):
-        if '{' in line:
-            return i + 1
-    return method_decl_start_line + 1
+    with open(file_path, encoding='utf-8') as f:
+        file_lines = f.read().split('\n')
+        for i, line in enumerate(file_lines[method_decl_start_line - 2:], method_decl_start_line - 2):
+            if '{' in line:
+                return i + 1
+        f.close()
```

Comment:
You don't need to close file if you use `with` construction.

Reply:
fixed